docs/ide/tutorial-csharp-console.md

```diff
@@ -56,11 +56,7 @@ If you don't see the **Console App (.NET Core)** project template, you can get i
 
 ## Create a "C# Console Calculator" app
 
-1. Open Visual Studio 2017, and then from the top menu bar, choose **File** > **New** > **Project**.
-
-1. In the **New Project** dialog box in the left pane, expand **C#**, and then choose **.NET Core**. In the middle pane, choose **Console App (.NET Core)**. Then name the file *Calculator*.
-
-1. Enter or paste the following code into the code editor:
+1. After creating a new **C# Console Application**, enter or paste the following code into the code editor:
```

Comment:
@kallerdaller This is a good edit to reduce the unintended duplication. However, to keep with our style of avoiding an "-ing" word when we can, and to add more clarity, could you please change this to "After you create the **C# Console Application**, enter or past the following code into the code editor:"

Reply:
@TerryGLee I have checked and I believe this is the last "-ing" word that I have introduced.